src/components/views/rooms/AuxPanel.js

```diff
@@ -80,6 +94,11 @@ module.exports = React.createClass({
         ev.preventDefault();
     },
 
+    _rateLimitedUpdate: new RateLimitedFunc(function() {
+        /* eslint-disable babel/no-invalid-this */
+        this.forceUpdate();
```

Comment:
the forceUpdate is a very bad smell. instead it should be `this.setState({'counters': <parse your counters out of state>})` or something, which then will automatically trigger the update.